### PR TITLE
[ticket/14265] Make all tables available in container

### PR DESCRIPTION
**Pull request by Elsensee**
[PHPBB3-14265](https://tracker.phpbb.com/browse/PHPBB3-14265)

### CODE REVIEW

File: phpBB/config/default/container/tables.yml [hunk 1]
```diff
@@ -1,28 +1,77 @@
...
+    tables.posts: %core.table_prefix%posts
+    tables.privmsgs: %core.table_prefix%privmsgs
+    tables.privmsgs_folder: %core.table_prefix%privmsgs_folder
+    tables.privmsgs_rules: %core.table_prefix%privmsgs_rules
+    tables.privmsgs_to: %core.table_prefix%privmsgs_to
     tables.profile_fields: %core.table_prefix%profile_fields
     tables.profile_fields_data: %core.table_prefix%profile_fields_data
     tables.profile_fields_options_language: %core.table_prefix%profile_fields_lang
     tables.profile_fields_language: %core.table_prefix%profile_lang
-    tables.posts: %core.table_prefix%posts
```
Comment (CHItA) [inline phpBB/config/default/container/tables.yml]: Why have you removed this?

Comment (Elsensee) [inline phpBB/config/default/container/tables.yml]: I feel better when this is in alphabetical order.

Comment (CHItA) [inline phpBB/config/default/container/tables.yml]: Oh, yeah sorry, i missed that. I should use search in the morning... :)

**Comment by Nicofuma** [comment]
Could you add a `dev:db:tables` (or something like that) console command showing a table with the defined `tables.*` parameters and the database tables?

**Comment by Nicofuma** [comment]
Sorry... wrong button

**Pull request merged**